src/Components/Input/index.css

```diff
@@ -0,0 +1,26 @@
+.InputContainer {
```

Comment:
So, I think the naming of the classes is not following `SUITCSS` conventions. Also, I wouldn't call the base class `InputContainer` since it might be confusing because this component is not an input container, it's an input.